### DISCUSSION TITLE
The list is always loading (Rclone/clouddrive)

### DISCUSSION THREAD

**Discussion by skyrim663** [Difegue/LANraragi#927] [Q&A]
**LRR Version and OS**
Docker latest(0.9.0),ubuntu20,Rclone+clouddrive

**Bug Details**
Similar to issue #858 , The list loads very slowly (35s on average) or loading until it times out.

The latest update solved the problem of not being able to open the Statistics page very well. However, loading the gallery list still loads indefinitely!
Since clouddrive generally has api read rate limitations, and the api quota does run out quickly after opening lanraragi. It's possible that the cloud storage api is overwhelmed by the frequent requests when loading the listings.
And the Modify Categories page cant open

**Comment by Difegue** [comment]
If you're storing the thumbnails on rclone as well it's likely that's where your timeouts are coming from that -- You could either use list view mode or move the thumbs somewhere else?
